src/react/ui/SingleCardPage.tsx

```diff
@@ -23,19 +23,29 @@ import BackButton from './BackButton';
 interface Props {
   backLink?: string;
   children: React.ReactNode;
+  contentCard?: boolean;
   title: string;
 }
 
-export default function SingleCardPage({ backLink, children, title }: Props): React.ReactElement {
+export default function SingleCardPage({
+  backLink,
+  children,
+  contentCard = true,
+  title,
+}: Props): React.ReactElement {
   return (
     <Container fluid className="p-3">
       <h2>
         {backLink && <BackButton href={backLink} />}
         {title}
       </h2>
-      <Container fluid className="bg-white rounded p-3 mt-4">
-        {children}
-      </Container>
+      {contentCard ? (
```

Comment:
I see why you're doing this here. But this is for single page cards. If your page has multiple cards, you shouldn't use this component

Reply:
That's right. I left SingleCardPage component untouched and hardcoded the layout directly into the page. I agree we can wait until all pages are converted into React, so we'll have a better understanding about how to extract a Page component that abstract common layout and style.